File: 3718_Smallest_Missing_Multiple_of_K/solution.c
int missingMultiple(int* nums, int numsSize, int k) {
    int n = k;
    while (1) {
        int found = 0;
        for (int i = 0; i < numsSize; i++) {
            if (nums[i] == k) {
                found = 1;
                break;
            }
        }
        if (!found) return k;
        k += n;
    }
}

File: 3718_Smallest_Missing_Multiple_of_K/solution.py
# class Solution:
#     def missingMultiple(self, nums: List[int], k: int) -> int:
#         n = k
#         while True:
#             if k not in nums:
#                 return k
#             else:
#                 k += n 

class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        nums_set = set(nums)
        n = k
        while k in nums_set:
            k += n
        return k